### ISSUE TITLE
Bug: Field option multiple dependencies not working properly across tabs in edit Pod UI

### ISSUE THREAD

**Issue by JoryHogeveen** [pods-framework/pods#3707]
See https://github.com/pods-framework/pods-seo/pull/6

**Use case:**

If you put two dependencies for a field in the tab "Advanced" that is dependent on a setting in the tab "Basic", this is not working nicely.
When opening the field these dependencies are not processed. Only when the actual dependency field is edited will the dependency functionality work.

Found this problem in the patch for Pods SEO where XML Sitemaps checkbox (Advanced tab) is only avaiable for the Field Type "_file_" (Basic tab) with File Types set to "_images_" (Additional Field Options tab).

When opening a different field type and going directly to the Advanced tab the XML Sitemaps option is visible. Only after clicking on the Field Type select box in the Basic tab is this changed according to dependencies.

**Use case 2:**
When adding a dependency that doesn't exist it validates as true instead of false. It seems more logical to me to make this false.

Example (also Pods SEO)
```php
'depends-on' => array(
    // This is a field type specific option (only file fields) 
    // It is considered true if it doesn't exists for a field
    'file_type' => 'images',

    // This is a Field Type dependency, available in all field types (works as expected)
    'type' => 'file',
),
```

**Comment by JoryHogeveen** [comment]
Did some extra research on this.

The issue happens because in this specific case the `file_type` is `images` by default. This means that the `file_type` dependency would be valid, but the `type` dependency isn't.

So if the field isn't a `file` field, this value is the default value and for some reason the dependency script thinks it's valid even though the first dependency (field type) is not `file`.

I'd say if the option has two dependencies, both of them must be valid, even if they reside in different tabs.

I'm having some trouble getting through the whole dependency script in `jquery.pods.js` (no documentation..). If anyone can shed some light here that would be great.

**Comment by JoryHogeveen** [comment]
Fixed in #3723
